Authentication/AuthenticationBootstrapper.swift

```diff
@@ -219,33 +235,29 @@ public class AuthenticationBootstrapper<User: UserType, SessionService: SessionS
 extension AuthenticationBootstrapper: LoginControllerTransitionDelegate {
     
     public func onSignup(controller: LoginController) {
```

Comment:
This one is a tricky one, in a regular app I would tell you that the name should communicate intention, so `onSignup` would be a better choice. But in this particular case I get your point. Maybe calling it `performTransitionToSignup`